empyric/adapters.py

```diff
@@ -1438,7 +1438,6 @@ def _query(self, *args, **kwargs):
     def disconnect(self):
         while self.connected:
             self.backend.close()
-            time.sleep(0.1)
```

Comment:
This seems to throw an error sometimes when called as the main thread is closing. This was originally put in to make sure that the disconnection process finishes, but is probably unnecessary.